packages/web/graphql/mutations/guild.ts

```diff
@@ -1,7 +1,7 @@
-import gql from 'fake-tag';
+export {};
```

Comment:
what exactly does this do?

Reply:
Since the file doesn't have any statements in it that is required for it to compile as a ESM module.

`yarn typecheck` told me I needed it.